### ISSUE TITLE
NTR: Fetal medication or drug exposure (Gestation & perinatal history)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7872]
New term request Fetal medication or drug exposure (f6): Gestational; prenatal

**Comment by pnrobinson** [comment]
@MickeySegal
I am adding this as a synonym to 
[Maternal exposure history HP:0034756](https://hpo.jax.org/app/browse/term/HP:0034756)
For each of the following "fetal" terms, I will add terms with both "Maternal X exposure" and synonym "Fetal X exposure"

I think the "Fetal X exposure" makes more sense as the primary label so I will adjust the existing terms and add the new ones you proposed.
